build.gradle.kts

```diff
@@ -62,6 +62,8 @@ tasks {
 }
 
 dependencies {
+
+  implementation("org.owasp:dependency-check-gradle:7.1.1")
```

Comment:
🙋 This is already included from `uk.gov.justice.hmpps.gradle-spring-boot`, so this change should not be needed. (The nightly job runs `./gradlew dependencyCheckAnalyze` without installing anything)

Reply:
Removed 👍